packages/npm-vue/README.md

```diff
@@ -0,0 +1,57 @@
+Reatom integration for Vue Composition API.
+
+## Installation
+
+```sh
+npm i @reatom/npm-vue
+```
+
+## API
+
+### `createReatomVue`
+
+A function that creates a [Vue App plugin](https://vuejs.org/guide/reusability/plugins.html#plugins) which you can use. Accepts a `Ctx` object or creates one if it's not provided.
+
+```ts
+import { createReatomVue } from '@reatom/npm-vue'
+
+app.use(createReatomVue())
+```
+
+### `reatomRef`
+
+A function that turns a Reatom atom into a Vue ref which is updated on target atom changes. A returned pseudo-ref is mutable if a target atom is mutable itself.
+
+Because all Reatom APIs require `ctx` to be available, you must either provide it with `createReatomVue` plugin or pass it explicitly as a second argument to `reatomRef`.
+
+## Example
+
+Setup `ctx` somewhere in the app root:
+
+```ts
+import { createCtx } from '@reatom/core'
+import { createReatomVue } from '@reatom/npm-vue'
+
+const ctx = createCtx()
+app.use(createReatomVue(/* optional */ ctx))
+```
+
+Then use Reatom state in your components:
+
+```ts
```

Comment:
Add an example with `.ts` file of the model (atoms) and `.vue` file with the usage